docs/DdnRelocatableShaderElf.md

```diff
@@ -0,0 +1,205 @@
+# Unlinked Shaders And Relocatable Shader Elf
+
+# INTRODUCTION
+
+There is interest in being able to compile SPIR-V shaders without having all of
+the pipeline state information available.  This is advantagous when the same
+shader is used in multiple pipelines because it will only be compiled once.  It
+is also possible to compile the shaders "offline", that is without running the
+full vulkan application. Unlinked and relocatable shaders were designed
+with the intend of allowing offline compilation of shaders that will be linked
+together during pipeline creation.
+
+# Unlinked shaders vs relocatable shaders
+
+The two concepts are similar, but there is a minor, and important, distinction.
+An unlinked shader is an LGC concept.  LGC will generate an unlinked shader
+when the option in the pipeline context is set. See `m_unlinked` in
```

Comment:
when [`m_unlinked`](https://github.com/GPUOpen-Drivers/llpc/blob/dev/llpc/context/llpcPipelineContext.h) in the pipeline context ...